### ISSUE TITLE
Pages missing #content ID

### ISSUE THREAD

**Issue by caitlynmayers** [caitlynmayers/dukes#134]
### Location
http://test-dukes.pantheonsite.io/category/event/
http://test-dukes.pantheonsite.io/category/news/
http://test-dukes.pantheonsite.io/chef/jason-alley/
http://test-dukes.pantheonsite.io/chef/sean-brock/
http://test-dukes.pantheonsite.io/chef/vivian-howard/
http://test-dukes.pantheonsite.io/chef/walter-bundy/
http://test-dukes.pantheonsite.io/stories/chefs/

### Description
The templates for the pages above are missing the #content id on the main content container that allows the skip nav link to work.

**Comment by caitlynmayers** [comment]
Fix verified.
